### PR TITLE
add contribex leads to label sync alert

### PR DESCRIPTION
**Pull request by BenTheElder**
follow up to https://github.com/kubernetes/test-infra/pull/32948

/assign @Priyankasaggu11929

### CODE REVIEW

File: config/jobs/kubernetes/sig-k8s-infra/trusted/sig-testing-label-sync.yaml [hunk 1]
```diff
@@ -38,5 +38,5 @@ periodics:
     testgrid-alert-stale-results-hours: '12'
     testgrid-dashboards: sig-testing-misc
     testgrid-tab-name: label_sync
-    testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
+    testgrid-alert-email: sig-contribex-leads@kubernetes.io kubernetes-sig-testing-alerts@googlegroups.com, k8s-infra-oncall@google.com
```
Comment (BenTheElder) [inline config/jobs/kubernetes/sig-k8s-infra/trusted/sig-testing-label-sync.yaml:41]: I think this is the right email? not kubernetes-sig-contribex-leads@googlegroups.com, which we don't have other references to here.

if it's not, then we need to update the tempelis alerts as well

Comment (Priyankasaggu11929) [reply config/jobs/kubernetes/sig-k8s-infra/trusted/sig-testing-label-sync.yaml]: yes, `sig-contribex-leads@kubernetes.io` is the correct email

**Comment by Priyankasaggu11929** [comment]
Thank you @BenTheElder :pray: 

/lgtm
/approve

File: config/jobs/kubernetes/sig-k8s-infra/trusted/sig-testing-label-sync.yaml (hunk 1 above)
Comment (BenTheElder) [reply config/jobs/kubernetes/sig-k8s-infra/trusted/sig-testing-label-sync.yaml]: Ok good, I think we've been using that consistently 😅

**Pull request merged**